docs/transactions/savepoints.html

```diff
@@ -6,7 +6,42 @@
   <#assign savepoints="active">
 </head>
 <body>
-<h2>SavePoints</h2>
+<h2>SavePoints (inside a transaction)</h2>
+
+<p>
+Ebean supports using <code>SavePoints</code> which are a feature of most Relational Databases
+in which you can create a "Save Point" inside a transaction to which you can rollback to in case of an error (or some other logic you want)
+and are exposed via the <code>JDBC</code> driver.
+</p>
+
+<p>
+If you have a batch job that needs to perform certain operations, but can fail at some point, but you don't want to loose all the work, <code>SavePoints</code> might be a tool for that. You can check more information on <a href='https://docs.oracle.com/javase/tutorial/jdbc/basics/transactions.html#set_roll_back_savepoints'>Oracle's JDBC tutorial</a>
+</p>
+
+Example of using a <code>SavePoint</code>
+```java
+  Customer newCustomer = new Customer();
+  newCustomer.setName("John");
+  Ebean.save(newCustomer);
+
+  try (Transaction transaction = Ebean.beginTransaction()) {
+
+    Connection connection = transaction.getConnection();
+    // Create a Save Point
+    Savepoint savepoint = connection.setSavepoint();
```

Comment:
Yes.  Note we also have built in support for savepoints via `transaction.setNestedUseSavepoint()`

See tests in: org.tests.transaction.TestNestedSubTransaction

```java
  @IgnorePlatform({Platform.SQLSERVER, Platform.MYSQL, Platform.HANA})
  @Test
  public void nestedUseSavepoint() {

    EbeanServer server = server();

    EBasic bean = new EBasic("start1");

    try (Transaction txn0 = server.beginTransaction()) {
      txn0.setRollbackOnly();
      txn0.setNestedUseSavepoint();

      server.save(bean);

      try (Transaction txn1 = server.beginTransaction()) {
        bean.setName("updateNested");
        server.save(bean);
        txn1.commit();
      }

      EBasic fresh = server.find(EBasic.class, bean.getId());
      assertNotNull(fresh);
      assertThat(fresh.getName()).isEqualTo("updateNested");

      try (Transaction txn2 = server.beginTransaction()) {
        bean.setName("barney");
        Ebean.save(bean);
        txn2.rollback();
      }

      fresh = server.find(EBasic.class)
        .setId(bean.getId())
        .setPersistenceContextScope(PersistenceContextScope.QUERY)
        .findOne();

      assertNotNull(fresh);
      assertThat(fresh.getName()).isEqualTo("updateNested");

    } finally {
      cleanup(bean);
    }
  }
```

Reply:
Ah, great, code-review for documentation 😅 👍 
I'll have to add that (once I understand it). Thanks Rob